ctparse/time/corpus.py

```diff
@@ -2,125 +2,125 @@
     # ruleYear
     ('Time[]{2019-X-X X:X (X/X)}',
      '2018-03-07T12:43',
-     ('2019',)),
```

Comment:
I haven't thought more than 5 seconds about it, but does it really make a difference to use a list over a tuple here?

Reply:
No, but mypy complained because of mixes between tuple and lists

Reply:
Oki, then lets change it....